plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -146,7 +147,7 @@ export const CatalogTable = (props: CatalogTableProps) => {
       return {
         icon: () => (
           <>
-            <Typography variant="srOnly">{title}</Typography>
+            <Typography style={visuallyHidden}>{title}</Typography>
```

Comment:
Switched these back to use `Typography` rather than `span`, figured it kinda makes sense and we avoid the lint rule skip